frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx

```diff
@@ -88,13 +88,68 @@ export function SummaryTable(): JSX.Element {
             title: 'Conversion rate',
             render: function Key(_, item): JSX.Element {
                 const conversionRate = conversionRateForVariant(experimentResults, item.key)
-                return (
-                    <div className="font-semibold">
-                        {conversionRate === '--' ? conversionRate : `${conversionRate}%`}
-                    </div>
-                )
+                if (!conversionRate) {
+                    return <>--</>
+                }
+
+                return <div className="font-semibold">{`${conversionRate.toFixed(2)}%`}</div>
             },
-        })
+        }),
+            columns.push({
+                key: 'delta',
+                title: (
+                    <div className="inline-flex items-center space-x-1">
+                        <div className="">Delta %</div>
+                        <Tooltip title="Delta % indicates the percentage change in the conversion rate between the control and the test variant.">
+                            <IconInfo className="text-muted-alt text-base" />
+                        </Tooltip>
+                    </div>
+                ),
+                render: function Key(_, item): JSX.Element {
+                    if (item.key === 'control') {
+                        return <></>
+                    }
+
```

Comment:
I know why we're returning nothing here (the control is the baseline), but IMO it feels a bit odd to have a table entry with _nothing_ in it, at first glance it makes me feel like there's a bug in the table.  Would it be possible to put a different visual indicator denoting that this cell represents the baseline?

Reply:
Good point, opted for this:

<img width="107" alt="image" src="https://github.com/user-attachments/assets/04201c36-97b4-4bb3-9e6e-5e880a342cd5">

(the dash empty state I want to only use in cases when there is no result now but may be later)